modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -1442,6 +1447,54 @@ describe("Token Voting Client", () => {
         const token = await client.methods.getToken(pluginAddress);
         expect(token).toBe(null);
       });
+
+      it("Should check if a ERC20 is compatible with governance and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc20Token = await deployErc20();
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
+            erc20Token.address,
+          )
+        ).rejects.toThrow(ERC165NotSupportedError)
+      });
+      it("Should check if ERC721 is compatible with governance and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc721Token = await deployErc721();
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
+            erc721Token.address,
+          )
+        ).rejects.toThrow(ERC20NotSupportedError);
+      });
+      it("Should be called with an invalid address and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
+            "0x123",
+          )
+        ).rejects.toThrow(InvalidAddressError);
+      });
+      it("Should be called with wallet invalid address and throw", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        expect(() =>
+          client.methods.isTokenGovernanceCompatible(
+            TEST_WALLET_ADDRESS,
+          )
+        ).rejects.toThrow(NotAContractError);
+      });
+      it("Should check if GovernanceERC20 is compatible with governance and return true", async () => {
```

Comment:
"and return true". Shouldn't it return an enum? Why it throws in the other cases?